app/models.py

```diff
@@ -1467,6 +1467,7 @@ def serialize(self):
         data.update({
             'id': self.id,
             'briefId': self.brief_id,
+            'briefTitle': self.brief.data.get('title', ''),
```

Comment:
This one feels a bit cheeky. On the one hand, it is no different from the `supplierName` that we are already including so maybe we are fine with it.

It feels a bit cheekier than adding `frameworkFramework` earlier to a `SupplierFramework` as the brief title is going to be used for presentation to a user rather than something like `frameworkFramework` which is likely not getting shown to the user but used in the frontend app logic.

I guess if I go with the convention from `supplierName` though I'm happy with it. Would be good to get a second opinion though to back that up if this is an approach we want to keep encouraging/doing more of for our API...